packages/waku/src/lib/plugins/vite-plugin-fs-router-typegen.ts

```diff
@@ -0,0 +1,194 @@
+import type { Plugin } from 'vite';
+import { promises, readFileSync, existsSync } from 'node:fs';
```

Comment:
Can we use `import { ... } from 'node:fs/promises'` ?

Reply:
Is this ok or did you not want to use sync calls at all? 

```ts
import { readdir, writeFile } from 'node:fs/promises';
import { existsSync, readFileSync } from 'node:fs';`
```

Reply:
that is ok.